src/index.html

```diff
@@ -8,6 +8,34 @@
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <header class="header" data-qa="card">
+      <img src="Rectangle 4.png"
+      class="header__img"
+      alt="picture">
```

Comment:
Fix it everywhere and change the image name to actual
```suggestion
      <img
        src="Rectangle 4.png"
        class="card__img"
        alt="IMac"
      >
```